dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2295,6 +2313,98 @@
         }
       }
     },
+    "ZeroShotClassificationTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Zero Shot Classification task.",
+      "properties": {
+        "classificationType": {
+          "type": "string",
+          "description": "Specifies the number of classes per document. Defaults to multi classification which may return more than one class for each document.",
```

Comment:
nit: This implies the type of the field is a numerical number of classes. I think we better rephrase to show that we have only either 1 or multiple classes but not an arbitrary given number.

Reply:
done